tests/functional/queries_with_sql_types.py

```diff
@@ -12,20 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import psycopg2 as pg
 import pytest
 
+from psycopg2._psycopg import connection, cursor
+from psycopg2 import connect, Error
+
+# all imports from errors are OK if you can find such exception class in docs
+from psycopg2.errors import NumericValueOutOfRange
+
 
 @pytest.fixture(scope="session", autouse=True)
-def create_cursor(request):
-    cur = None
-    conn = None
+def create_cursor(request) -> cursor:
 
-    conn = pg.connect(host="localhost", password="check_this_out", database="postgres")
-    assert conn is not None
+    # ToDo - connection process test is not ideal yet.
```

Comment:
Could you please create `Feature request` issue with description how connection process can be improved?

Reply:
It is not ideal for my opinion, but it can be ideal already. I don't know how to make it better right now, but will work on it.